src/tools/cephfs/cephfs-shell

```diff
@@ -1315,6 +1315,14 @@ Access: {}\nModify: {}\nChange: {}".format(path.decode('utf-8'), stat.st_size,
                 self.perror('{}: no such file or directory'.format(path.decode('utf-8')),
                             end='\n', apply_style=True)
 
+def read_ceph_conf(config_file):
+    from configparser import ConfigParser
+
+    cp = ConfigParser(strict=False)
+    cp.read(config_file)
+    if cp.__contains__('cephfs-shell'):
+        for option in cp.options('cephfs-shell'):
+            setattr(shell, option, cp['cephfs-shell'][option])
```

Comment:
Ceph configurations are managed by the monitors. They may also be sourced from a local config file. You should use `libcephfs.conf_get`.

Reply:
Ok. I'll do that instead.

Reply:
Thanks! This issue was blocking this PR for a long time.